nemo_rl/data/datasets/response_datasets/conversation_base.py

```
@@ -0,0 +1,131 @@
+        message["value"] = msg_str
+        data["conversations"][idx] = message
+
+    if not return_inplace:
+        return data
+
+
+def conversation_process_message(
+    metadata: Dict[str, Any],
+    message: Dict[str, str],
+    media_index: dict,
+    raw: Dict[str, Any] = {},
```

Ruff (0.14.8)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)